### ISSUE TITLE
xxx.txt in test file，prepare_data

### ISSUE THREAD

**Issue by Zmeng830** [xyjbaal/fpcc_pytorch#4]
1. I want to know how the pytorch version of the code configures the IPA and XA data sets. I generated the .h5 file for training and the .txt file containing the .h5 file through the four .py files of the tensorflow version. Then put these files into the datas folder of the pytorch version and run fpcc-train training. I got the training results of epoch_1~250.pt, which are saved in the ring_vdm_asm folder. 
2. But I don’t know how to prepare the test file. For example, where does the .txt file come from under the ring_test file?
3. Are the meta and _data/util/h5.py files in the prepare_data folder in the torch version code redundant? It feels like it's about target detection. It is difficult to apply the IPA dataset to these .py files.

**Comment by xyjbaal** [comment]
I use .txt files in test instead of .h5 files,

Please refer to these files.
[datas/ring_test](https://github.com/xyjbaal/FPCC/tree/844c40c5ce70e16962d994c040b19592ac233842/datas/ring_test)

**Comment by Zmeng830** [comment]
Thank you very much for your reply,
After consulting you last time, I used IPA_image2pc.py to generate xx.txt file from the testing dataset in [IPA](https://owncloud.fraunhofer.de/index.php/s/AacICuOWQVWDDfP) for testing. I don't know if I am doing this right?
I would like to ask the file you provided for testing,such as  026.txt , which has 5 columns. The xx.txt file I generated above has 4 columns, but I didn’t find in the paper or code how you processed it to get 5 columns in the file you provided. Maybe this seems to have no effect on testing?
Sorry to bother you again！
the file you provided
-0.073684 -0.274834 1.416667 0.261808 11.000000
-0.072005 -0.272552 1.404902 0.261808 11.000000
the file i generate
-0.072868 -0.271791 1.400980 11.000000
-0.071603 -0.271030 1.397059 11.000000
